### PR TITLE
increase stack size on windows

### PR DESCRIPTION
**Pull request by Zeegomo**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -126,6 +126,10 @@ jobs:
           esac
           echo "JORMUNGANDR_FEATURES=$features" >> $GITHUB_ENV
 
+          if [ '${{ matrix.profile }}' == debug ] && [ '${{ matrix.os }}' == windows-latest ]; then
+            echo "RUSTFLAGS=$RUSTFLAGS -C link-args=/STACK:2000000" >> $GITHUB_ENV
```
Comment (enzoc4) [inline .github/workflows/test.yml:130]: is this twice the current stack size?

**Review by enzoc4 — APPROVED** [review]
not sure what's this for, are we really overflowing the stack in tests?

But anyway, I think this is probably ok, it's a small number anyway.

File: .github/workflows/test.yml (hunk 1 above)
Comment (Zeegomo) [reply .github/workflows/test.yml:130]: So, yes, we were overflowing the stack size on windows for debug builds in certain edge cases (https://github.com/input-output-hk/jormungandr/runs/4852642457?check_suite_focus=true).

Looking at the binary with `editbin` it looks like the current value was 100k, which is quite small, especially since the default should be 1 MB (it's 8 MB on linux for comparison). Not sure of why it was that small though

**Review by Mr-Leshiy — APPROVED** [review]
